### PR TITLE
Update ruboco and rubocop-rspec, then pin their versions

### PR DESCRIPTION
**Pull request by awead**
This should keep us insulated from Rubocop's whimsical nature.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 require: rubocop-rspec
 
+inherit_from: .rubocop_todo.yml
+
 AllCops:
-  RunRailsCops: true
```
Comment (mjgiarlo) [inline .rubocop.yml]: Are the Rails cops enabled by default?

Comment (awead) [inline .rubocop.yml]: Dunno. That option is no longer valid.

**Comment by awead** [comment]
ping @coblej @barmintor this should get the gem passing, but you'll note that current versions of hydra-head and active-fedora aren't supported.

File: .rubocop.yml (hunk 1 above)
Comment (mjgiarlo) [inline .rubocop.yml]: The incantation changed to:

```yaml
Rails:
  Enabled: true
```

**Comment by barmintor** [comment]
ok I cleaned up the commits a bit. The tests were hung up on lingering class declarations in the specs sometimes, depending on order of execution. This PR is passing mri 2.2.0, 2.2.1, and 2.3.0; it is running with current AF.

**Comment by awead** [comment]
Looks good, although why test 2.2.0 and 2.2.1? We could just support 2.3 and encourage folks to update their rubies. Also, can you squash up the commits? That first one is particularly bad. :smile:

**Comment by barmintor** [comment]
I'm happy to remove 2.2.0 and squash, though I like keeping the travis
config changes in their own commits personally. Do you want me to squash
the others?

**Comment by awead** [comment]
@barmintor yeah, that'd be great. I'm happy to merge it once that's done.

**Pull request merged**